pg-index-health-spring-boot-kotlin-demo/src/main/kotlin/io/github/mfvanek/pg/index/health/demo/kotlin/config/ForeignKeyJsonDeserializer.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019-2025. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health-demo
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+package io.github.mfvanek.pg.index.health.demo.kotlin.config
+
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.core.ObjectCodec
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonDeserializer
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.TextNode
+import io.github.mfvanek.pg.model.column.Column
+import io.github.mfvanek.pg.model.constraint.ForeignKey
+import org.springframework.boot.jackson.JsonComponent
+import java.io.IOException
+
+@JsonComponent
+class ForeignKeyJsonDeserializer : JsonDeserializer<ForeignKey>() {
```

Comment:
Take a look at https://github.com/mfvanek/pg-index-health-demo/blob/master/pg-index-health-spring-boot-demo/src/main/java/io/github/mfvanek/pg/index/health/demo/config/JacksonConfig.java

Now we can get rid of this class